lib/matplotlib/markers.py

```diff
@@ -369,7 +390,10 @@ def get_transform(self):
         Return the transform to be applied to the `.Path` from
         `MarkerStyle.get_path()`.
         """
-        return self._transform.frozen()
+        if self._user_transform is None:
+            return self._transform.frozen()
+        else:
+            return (self._transform + self._user_transform).frozen()
```

Comment:
Codecov claims that this line isn't being hit by the tests - I'm not sure why that's the case? Does that mean it's redundant?

Reply:
I believe we need this. There are plenty of cases, where `get_transform()` is called. However, in those cases, no user supplied transforms are used. I have added specific test that compares that plain marker `get_transform()` and marker with supplied transform returns appropriate results.